pyiron_atomistics/atomistics/job/interactive.py

```diff
@@ -156,7 +156,7 @@ def interactive_index_organizer(self):
             self.interactive_indices_setter(self._structure_current.indices)
 
     def interactive_cell_organizer(self):
-        if not np.allclose(
+        if self._generic_input["calc_mode"] != 'static' or not np.allclose(
```

Comment:
This is not necessary for `calc_minimize()` only `calc_minimize(pressure=0)` right? And analogously it is not necessary for `nvt` and `nve` but `npt`.

Reply:
Is the information about `nvt`, `nve` etc.  included in the input?

Reply:
Not yet but we could include it in https://github.com/pyiron/pyiron_atomistics/blob/master/pyiron_atomistics/atomistics/job/atomistic.py#L226

Reply:
So we could go ahead with this version for now, right? I don't think communicating the cell size would significantly change something.